### PR TITLE
v0.26.0 -> Master

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Preparing a batch of changes to release 0.26 without locking an ability to release patch versions for 0.25

Thanks

### CODE REVIEW

File: DEPRECATION.md [hunk 1]
```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
...
-[link-hearbeatinterval]:
-  https://cube.dev/docs/@cubejs-client-ws-transport#web-socket-transport-hear-beat-interval
+### Embedding Cube.js within Express
 
-### Node.js 8
+**Deprecated in Release: v0.24.0**
 
-**Deprecated in Release: v0.22.4**
+Embedding Cube.js into Express applications is deprecated due to performance and
+reliability considerations. [Read more about this change
```
Comment (rpaik) [inline DEPRECATION.md:86]:
```suggestion
reliability issues. [Read more about this change
```

Comment (rpaik) [reply DEPRECATION.md:86]: Is "issues" too strong?

Comment (hassankhan) [reply DEPRECATION.md:86]: Possibly ... this PR is specifically related to changes for v0.26 though 👍

File: DEPRECATION.md [hunk 2]
```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
...
+[link-migration]:
+  https://cube.dev/docs/configuration/overview#migrating-from-express-to-docker
 
-### Absolute import for @cubejs-backend/query-orchestrator
+### Absolute import for `@cubejs-backend/query-orchestrator`
 
 **Deprecated in Release: v0.24.2**
 
-Reason: Absolute imports highly depend on a path, and all API becomes public. Now we started to provide public API as `export` from the module.
+Absolute imports highly depend on a path, and all API becomes public. We now
```
Comment (rpaik) [inline DEPRECATION.md:101]: How about "Absolute imports are highly dependent on a path, "?

File: DEPRECATION.md [hunk 3]
```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
...
+
+[link-contexttoorchestratorid]:
+  https://cube.dev/docs/config#options-reference-context-to-orchestrator-id
+
+### Absolute import for `@cubejs-backend/server-core`
 
 **Deprecated in Release: v0.25.4**
 
-Reason: Absolute imports highly depend on a path, and all API becomes public. Now we started to provide public API as `export` from the module.
+Absolute imports highly depend on a path, and all API becomes public. We now
```
Comment (rpaik) [inline DEPRECATION.md:134]: Same comment as above.

File: DEPRECATION.md [hunk 4]
```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
...
 const { CubejsServerCore } = require('@cubejs-backend/server-core');
 ```
 
-### Absolute import for @cubejs-backend/schema-compiler
+### Absolute import for `@cubejs-backend/schema-compiler`
 
 **Deprecated in Release: v0.25.21**
 
-Reason: Absolute imports highly depend on a path, and all API becomes public. Now we started to provide public API as `export` from the module.
+Absolute imports highly depend on a path, and all API becomes public. We now
```
Comment (rpaik) [inline DEPRECATION.md:153]: Same comment as above.

File: DEPRECATION.md (hunk 2 above)
Comment (hassankhan) [reply DEPRECATION.md:101]: Not sure, I think it's better to keep the API part but perhaps reword it to the following:

```suggestion
Absolute imports highly depend on a path, and their API becomes public. We now
```

File: DEPRECATION.md [hunk 5]
```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
...
+no more updates. Please upgrade to Node.js 12 or higher.
+
+[link-nodejs-eol]: https://github.com/nodejs/Release#end-of-life-releases
+
+### `USER_CONTEXT`
+
+**Deprecated in Release: v0.26.0**
+
+There are three names for effectively the same concept, and we wanted to take
+the opportunity to simplify this.
```
Comment (hassankhan) [inline DEPRECATION.md:195]: Forgot to remove these

File: DEPRECATION.md [hunk 6]
```diff
@@ -33,30 +33,50 @@ features:
...
-| Deprecated | Absolute import for @cubejs-backend/query-orchestrator               | v0.24.2    | v0.28.0      |
-| Deprecated | Absolute import for @cubejs-backend/server-core                      | v0.25.4    | v0.30.0      |
-| Deprecated | Absolute import for @cubejs-backend/schema-compiler                  | v0.25.21   | v0.32.0      |
+| Status     | Feature                                                                                                            | Deprecated | Remove       |
+| ---------- | ------------------------------------------------------------------------------------------------------------------ | ---------- | ------------ |
+| Deprecated | [Node.js 8](#nodejs-8)                                                                                             | v0.22.4    | v0.26.0      |
+| Deprecated | [`hearBeatInterval`](#hearbeatinterval)                                                                            | v0.23.8    | June 2021    |
+| Deprecated | [`CUBEJS_ENABLE_TLS`](#cubejs_enable_tls)                                                                          | v0.23.11   | v0.26.0      |
```
Comment (hassankhan) [inline DEPRECATION.md:40]:
```suggestion
| Removed    | [`CUBEJS_ENABLE_TLS`](#cubejs_enable_tls)                                                                          | v0.23.11   | v0.26.0      |
```

File: DEPRECATION.md [hunk 7]
```diff
@@ -33,30 +33,50 @@ features:
...
+
+This option for [`@cubejs-client/ws-transport`][link-hearbeatinterval] has been
+replaced by `heartBeatInterval`.
+
+[link-hearbeatinterval]:
+  https://cube.dev/docs/@cubejs-client-ws-transport#web-socket-transport-hear-beat-interval
+
+### `CUBEJS_ENABLE_TLS`
+
+**Deprecated in Release: v0.23.11**
```
Comment (hassankhan) [inline DEPRECATION.md:72]:
```suggestion
**Removed in Release: v0.26.0**
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Minor things that need a quick fix. 

@ovr would you prefer I make the changes myself as a PR to v0.26?
